### ISSUE TITLE
Allow devs to specify that a new IA is just an idea

### ISSUE THREAD

**Issue by GuiltyDolphin** [duckduckgo/community-platform#1243]
When creating a new IA page (or even once it is created), you should have the ability to make it clear that the IA is just an idea that you want someone else to develop, this might help to clear up some confusion on whether people are actively developing the IAs.

In addition to that - maybe the creator can *assign* someone to work on it (maybe that person then has to 'agree' before it displays) and then that could be used to show whether the IA has an active developer.

/cc @zekiel

**Comment by zekiel** [comment]
@GuiltyDolphin  we've been considering the best way to balance ideas, development, and responsibility. Unfortunately, our previous ideas forums were crowded w/ ideas that just plainly couldn't be done as an Instant Answer but we're first trying to optimize the flow for: 

1. I want to make DuckDuckGo the best search engine for ____topic_I_care_about____ 
2. I can contribute to those queries with Instant Answers that cover them. 

Essentially, IA pages are (currently) supposed to be the full blueprint for what you're actively working on so that others can follow, help, etc. It's explicit intent to develop an IA and maintain the queryspace. 

The IA pages could have an assignment feature (something we're hoping to introduce to Issues soon, since GitHub now allows assignment to team members w/ read access to a repo) so definitely open to thoughts around how the overall governance/process would work!

**Comment by MariagraziaAlastra** [comment]
@GuiltyDolphin did you see the [dev pipeline](https://duck.co/ia/dev/pipeline)?
The IA Pages in the Planning stage are nothing more than ideas, until they have a PR and get moved to Development. 

When a PR is merged the person who worked on it gets added to the contributors on the IA Page, and also contributors can be edited by admins, so if someone is interested in your IA Page he/she can ask us whether it's free to be taken.

It's true that we don't have a good way of saying "this IA is open to anyone who wants to develop it", though, although it doesn't really happen a lot either: usually the creator is the one who's going to actually work on it. 

Thanks for bringing this up and for the suggestions!

**Comment by GuiltyDolphin** [comment]
@MariagraziaAlastra Yeah, I've seen the dev pipeline; I was thinking more about your third paragraph ("this IA is open to anyone...") as I've come across cases where the developer hadn't gotten round to making a PR yet and someone else did.
